ProcessList.c

```diff
@@ -18,6 +18,7 @@ in the source distribution for its full text.
 #include "Platform.h"
 #include "Vector.h"
 #include "XUtils.h"
+#include "linux/LinuxProcess.h"
```

Comment:
Unnecessary import …

Apart from this import being forbidden in the (platform-independent) core code …

Reply:
This import is removed.